gitman/tests/test_shell.py

```diff
@@ -63,21 +64,25 @@ def test_ln(self, mock_call):
     def test_ln_missing_parent(self, mock_call):
         """Verify the commands to create symbolic links (missing parent)."""
         shell.ln('mock/target', 'mock/source')
-        assert_calls(mock_call, ["ln -s mock/target mock/source"])
+        assert_calls(mock_call, ["mkdir -p mock",
```

Comment:
I would expect this test two have two different results based on `os.name == 'nt`.